erpnext_indonesia_localization/erpnext_indonesia_localization/doctype/tax_invoice_exporter/tax_invoice_exporter.py

```diff
@@ -509,17 +509,16 @@ def process_csv_row_non_pkp(self, writer):
 					prefix_code = get_tax_prefix_code(si_doc)
 					tin_doc = frappe.get_doc("Tax Invoice Number", invoice_row.tax_invoice_number)
 					cust_doc = frappe.get_doc("Customer", invoice_row.customer)
-					cust_doc_address = frappe.get_doc("Address", f"{cust_doc.name}-Shipping")
 
 					self.validate_customer_tax_detail(cust_doc)
 
-					customer_address = getattr(cust_doc, 'company_address_tax_id', None) or \
-									   getattr(cust_doc, 'primary_address', None) or \
-									   getattr(cust_doc_address, 'address_line1', None)
-
-					if customer_address is None:
+					customer_address = None
+					if cust_doc.company_address_tax_id:
+						customer_address = cstr(cust_doc.company_address_tax_id)
```

Comment:
You can assign value of `customer_address` in line 522 to this variable to shorten the code